### DISCUSSION TITLE
Wacky Worlds Scenario Order needs to be different for those playing via RCTC

### DISCUSSION THREAD

**Discussion by HtotheTML** [OpenRCT2/OpenRCT2#22067] [Feature requests]
For some reason, exactly one RCT2 scenario had its objective changed for RCTC.

Mirage Madness (Africa - Oasis) originally had a goal of 1500 guests by the end of year 5.
In Classic, this was upped to 2100 guests year 5.

At present, Mirage Madness is the first challenging scenario, being placed at spot 6.
If at all possible, there needs to be a way to have the scenario list modified to move Mirage Madness further down on the list when playing with RCTC.

**Comment by Gymnasiast** [comment]
> If at all possible, there needs to be a way to have the scenario list modified to move Mirage Madness further down on the list when playing with RCTC.

This is not really possible, I’m afraid. (Or well, anything’s possible, but it wouldn’t be worth it.)
